Mantle/MTLModel.m

```diff
@@ -283,7 +323,7 @@ - (BOOL)validate:(NSError **)error {
 #pragma mark NSCopying
 
 - (instancetype)copyWithZone:(NSZone *)zone {
-	return [[self.class allocWithZone:zone] initWithDictionary:self.dictionaryValue error:NULL];
+	return [[self.class allocWithZone:zone] initWithValidatedDictionary:self.dictionaryValue error:NULL];
```

Comment:
What if we instead did something like

```objc
- (instancetype)copyWithZone:(NSZone *)zone {
    MTLModel *copy = [[self.class allocWithZone:zone] init];
    [copy setValuesForKeysWithDictionary:self.dictionaryValue];
    return copy;
}
```

?

Since the initializer is private and there is no way to return an error through `-copy`, catching `-setValue:forKey:` exceptions (that shouldn't occur anyway) makes little sense.